backends-clickhouse/src/test/scala/org/apache/gluten/execution/tpch/GlutenClickHouseTPCHSaltNullParquetSuite.scala

```diff
@@ -3430,5 +3431,12 @@ class GlutenClickHouseTPCHSaltNullParquetSuite extends GlutenClickHouseTPCHAbstr
       compareResultsAgainstVanillaSpark(sql3, true, checkOnlyOneAggregate)
     }
   }
+
+  test("GLUTEN-9177: Fix diff of parse_url") {
+    val select_sql = "select id, parse_url('http://user:pass@locahost', 'HOST'), " +
```

Comment:
const expression is still evaluated in vanilla spark although gluten is enabled.